src/renderer/components/team/members/MemberLogsTab.tsx

```
@@ -0,0 +1,243 @@
+              ? `subagent:${log.sessionId}:${log.subagentId}`
+              : `lead:${log.sessionId}`)
+              ? detailChunks
+              : null
+          }
+          detailLoading={
+            expandedId ===
+              (log.kind === 'subagent'
+                ? `subagent:${log.sessionId}:${log.subagentId}`
+                : `lead:${log.sessionId}`) && detailLoading
+          }
+          onToggle={() => void handleExpand(log)}
```

ESLint

[error] Remove this use of the "void" operator.

(sonarjs/void-use)